.gitignore

```diff
@@ -13,6 +13,8 @@ requests.egg-info/
 *.pyc
 *.swp
 *.egg
+.eggs/
+.tox/
 env/
 .venv/
 .eggs/
```

Comment:
Why are you duplicating `.eggs` here? And `.tox` below?

Reply:
ah jeez, let me fix that...

Reply:
hrm, I amended the commit and force-pushed on my fork, but this change is still showing up...

Reply:
ok here goes.

Reply:
GitHub does that to avoid confusing folks with "I left feedback here! They force-pushed and now it's disappeared!"